frontend/src/apiMocks/handlers.ts

```diff
@@ -183,6 +184,7 @@ export function getHandlers(
     const id = (ownAddresses[ownAddresses.length - 1]?.id ?? -1) + 1;
     ownAddresses.push({
       address: body.address ?? `custom_alias_${id}`,
+      full_address: body.address ?? `custom_alias_${id}`,
```

Comment:
Shouldn't ``full_address`` have a `@domain` suffix?

Also?
![Screen Shot 2022-03-31 at 3 30 47 PM](https://user-images.githubusercontent.com/286017/161146960-1abf9d1f-525d-482b-8ba9-e1f7563c544b.png)

![Screen Shot 2022-03-31 at 3 31 53 PM](https://user-images.githubusercontent.com/286017/161144268-c88a7a9b-3389-41aa-8753-b1d8e6e7b015.png)

![Screen Shot 2022-03-31 at 3 31 22 PM](https://user-images.githubusercontent.com/286017/161144328-8ed184ed-0ca4-4d15-83dd-b8d93b7dbb18.png)

Are you pranking me?

(from https://www.becomebetterprogrammer.com/typescript-double-question-marks/, which is a pretty good explanation for this Python dev)

Reply:
Oh btw, the custom domain used in the API mocks ([example](https://fx-relay-demo.netlify.app/?mockId=full)) is `mydomain.mozmail.com`.